app/src/main/res/layout/activity_create_tag.xml

```diff
@@ -73,46 +74,62 @@
         app:layout_constraintTop_toBottomOf="@+id/tagDescription"
         app:srcCompat="@android:drawable/ic_menu_report_image" />
 
-    <Button
+    <ImageButton
         android:id="@+id/takeTagPictureBtn"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
+        android:layout_width="90dp"
+        android:layout_height="50dp"
         android:layout_marginStart="128dp"
         android:layout_marginTop="30dp"
         android:layout_marginEnd="129dp"
         android:background="@drawable/rounded"
+        android:src="@drawable/ic_action_take_photo"
         android:fontFamily="monospace"
         android:padding="10dp"
-        android:text="@string/take_tag_picture"
-        android:textColor="@color/white"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/tagImage" />
 
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:layout_constraintTop_toBottomOf="@+id/takeTagPictureBtn"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintLeft_toLeftOf="parent"
+        android:layout_marginTop="40dp"
+        android:text="This tag will expire on March 20, 2020"
```

Comment:
Do we need this placeholder text if the tag's expiration is set programmatically?

Reply:
just for when I design the layout to see what it would look like. Will get rid of it